pages/docs/deployment.mdx

```diff
@@ -31,115 +31,19 @@ curl -L https://www.grouparoo.com/docker-compose --output docker-compose.yml
 docker-compose up
 ```
 
-## Run Locally with Node.js
-
-### Step 1: Install Runtime and Databases
-
-Grouparoo is a [node.js](https://nodejs.org/) application and requires Node.js version 12 or higher. You can download Node.js from their [website](https://nodejs.org/) or through the package manager for your operating system. The same is true for Postgres and Redis.
-
-We also need to create a database within Postgres for Grouparoo to use. Let's use one called `grouparoo_development`.
-
-**On OSX**:
-
-```bash
-# Install Homebrew (https://brew.sh) if you don't have it
-/bin/bash -c "$(curl -fsSL https://raw.githubusercontent.com/Homebrew/install/master/install.sh)"
-
-brew install node # Install Node.JS
-
-brew install redis # Install Redis
-brew services start redis # Start Redis
-
-brew install postgresql # Install Postgres
-brew services start postgresql # Start Postgres
-createdb grouparoo_development # Create a database for Grouparoo
-```
-
-**On Linux/Ubuntu**:
-
-```bash
-# Install NVM
-curl -o- https://raw.githubusercontent.com/nvm-sh/nvm/v0.35.3/install.sh | bash
-nvm install v12 # install v12 of node.js
-nvm use v12 # use v12 of node.js
-
-sudo apt install redis-server # Install Redis
-
-sudo apt install postgresql postgresql-contrib # Install Postgres
-createdb grouparoo_development # Create a database for Grouparoo
-```
-
-### Step 2: Generate `package.json` and `.env`
+Continue with installation details in our Docker installation guide.
 
-In a new directory, run the following command to generate the files Grouparoo needs:
+<Button variant="outline-primary" href="/docs/deployment/docker">
+  Get Started with Docker
+</Button>
```

Comment:
I think this is OK for simplicity.  Next would want you to do something like `<Button onClick={router.push("/docs/deployment/docker")}/>` for the fancy hot-reloading page changes.

Reply:
[I added a story to capture this](https://www.pivotaltracker.com/n/projects/2401683/stories/176786039).